spotdl/providers/audio/base.py

```diff
@@ -109,7 +109,6 @@ def __init__(
             "cookiefile": self.cookie_file,
             "outtmpl": str((get_temp_path() / "%(id)s.%(ext)s").resolve()),
             "retries": 5,
-            "extractor_args": [],
```

Comment:
```suggestion
        }
            "extractor_args": {},
```
discussion in thread